dev_apps/first-actor/choose.md

```diff
@@ -0,0 +1,25 @@
+# Choose your adventure
+
+To demonstrate how `wasmcloud` leverages the powerful polygot nature of webassembly, we have provided you with three identical calculators, written in three different languages.  Throughout the tutorial, you will see sections similiar to this one:
```

Comment:
don't use `wasmcloud`, instead just "wasmCloud" as a word. 
**WebAssembly** is always written with a capital W and capital A.

typo: spelled polyglot wrong.